src/Datadog.Trace.ClrProfiler.Managed/Helpers/HttpBypassHelper.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Datadog.Trace.ClrProfiler.Helpers
+{
+    internal static class HttpBypassHelper
+    {
+        public static bool ShouldSkipResource(Uri requestUri, string[] patternsToSkip)
```

Comment:
The `requestUri` parameter should probably be a string, since we don't need any of the `Uri` functionality. Callers that have uris can call `ToString()` themselves, and callers that have strings (like the unit tests) don't need to create a throwaway uri.

Reply:
I ended up passing a Uri so that the ToString call can be deferred if we don't need it.

Reply:
Shouldn't we avoid the call to `HttpBypassHelper.ShouldSkipResource()` entirely if we don't need it?

Reply:
What do you suggest? I was thinking about a strategy pattern that swaps out the ShouldSkipResources method with a return false methods if the config isn't set, but ran into some oddities where the settings belong to the Tracer instance, so doing so ends up meaning the Tracer needs to carry along it's own HttpBypassHelper and it was a big rabbit hole....

So, do you think this method should check the patterns and split paths or what?